ui/components/multichain/network-manager/components/custom-networks/custom-networks.tsx

```diff
@@ -130,20 +133,19 @@ export const CustomNetworks = React.memo(() => {
     });
 
     return filteredNetworks.length > 0 ? (
-      <>
+      <Box paddingBottom={2}>
```

Comment:
Added padding so that the custom networks, and testnets have some breathing room to match figma.